src/scss/layout/_title.scss

```diff
@@ -52,3 +52,16 @@
     }
   }
 }
+
+span.first {
```

Comment:
These should probably be specific to spans in `#title`? Unless we expect to use them elsewhere?